src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -240,6 +240,8 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
 
       <Jump id="marketSignalsTop" />
 
+      <Spacer y={2} />
```

Comment:
This winds up being a weird one. Actually the about tab has the same issue; the reason being is that the actual parent — the app with tabs: https://github.com/artsy/force/blob/4331d87e14354f15419e682ec609f9e897d65d8f/src/Apps/Artist/ArtistApp.tsx#L68 doesn't provide any margin on mobile but does on desktop. This seems to be because of the filter bar structure on mobile requiring 0 margin.

So to go forward with this:

I would change this to `<Spacer y={[2, 0]} />` — margins collapse so it's technically not a problem on desktop but just trying to be a bit more explicit about what we are targeting maybe (if, for instance this margin were to ever exceed `4`.) And then add this same thing to the about tab.

This winds up kind of breaking the 'no external margins' rule since this one dangles off the top here. I think it's OK in this instance because this is a route, not a component.

Reply:
Oh wow. Good spot for the about tab! I was testing it always against Artworks 😅